app/controllers/application_controller.rb

```diff
@@ -12,7 +12,9 @@ def status
   private
 
   def fetch_pages
-    @pages = Page.all # For side menu
+    # List pages for side menu
+    @pages = Page.where(language: I18n.locale)
```

Comment:
Keep in mind that an index on `language` would be useful for a large number of pages and languages.
IMO it is not needed for the current amount of pages.